### PR TITLE
Bug

### PR DESCRIPTION
**Pull request by iandioch**

### CODE REVIEW

**Comment by devoxel** [comment]
good one @iandioch

File: steely/plugins/bug.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+'''
+.bug <bug title> - <bug descripsion>
+.bug info <bug title>
+.bug list
+
+bug titles can't be longer than 20 chars
+bug descriptions can't be longer than 1000 chars or less than 20 chars.
```
Comment (iandioch) [inline steely/plugins/bug.py]: but why

File: steely/plugins/bug.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+'''
+.bug <bug title> - <bug descripsion>
```
Comment (itsdoddsy) [inline steely/plugins/bug.py]: - descrip**s**ion
+ descrip**t**ion

File: steely/plugins/bug.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+    if s is None:
+        return "no bug found"
+
+    return f'#{s.doc_id}: {s["title"]}\n{s["desc"]}'
+
+def list_bugs(_):
+    o = []
+    for item in BUG_DB:
+        o.append(f'{item.doc_id}:\t{item["title"]}')
```
Comment (iandioch) [inline steely/plugins/bug.py]: Wouldn't a list comprehension be nicer

File: steely/plugins/bug.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    return formatting.code_block('\n'.join(o))
+
+def add_bug(m):
+    if len(m) < 2:
+        return ERR_MSG
+
+    title, desc = m[0], m[1]
+    if len(title) > 15 or len(desc) > 1000 or len(desc) < 20:
```
Comment (iandioch) [inline steely/plugins/bug.py]: 😡

**Comment by iandioch** [comment]
((approval))

**Comment by devoxel** [comment]
@byxor Fixed the variable names, but can't test it at the moment.

**Pull request merged**